README.md

```diff
@@ -193,6 +193,38 @@ In the case one of your dependencies is installing `babel` and you cannot uninst
   }
 ```
 
+## Customize config for specific Webpack target
+
+Webpack supports bundling for multiple different [targets](https://webpack.js.org/concepts/targets/) including the web, and node. Sometimes it can be useful to have a slightly different babel configuration for the different environments. This loader provides a `target` property via the babel [caller](https://babeljs.io/docs/en/config-files#apicallercb) API which can be used in this case:
+
+```javascript
+// babel.config.js
+
+module.exports = api => {
+  return {
+    plugins: [
+      "@babel/plugin-proposal-nullish-coalescing-operator",
+      "@babel/plugin-proposal-optional-chaining"
+    ],
+    presets: [
+      [
+        "@babel/preset-env",
+        {
+          useBuiltIns: "entry",
+          // caller.target will be the same as the target option from webpack
+          ...(api.caller(caller => caller.target === "node")
+            ? { node: "current" }
```

Comment:
Shouldn't this also be `{ targets: { node: 'current' }}`?

Reply:
Yeup, you're right. I've gone ahead and fixed that up. Thanks!